src/abp-filter-parser.js

```diff
@@ -210,14 +210,7 @@ export function parseFilter(input, parsedFilterData, bloomFilter, exceptionBloom
   parsedFilterData.isRegex = input[beginIndex] === '/' &&
     input[input.length - 1] === '/' && beginIndex !== input.length - 1;
   if (parsedFilterData.isRegex) {
-      
-      // check for escaped forward slashes
-      if (input.match(/\/$/)) {
-          parsedFilterData.data = input.slice(beginIndex + 1);
-      } else {
-          parsedFilterData.data = input.slice(beginIndex + 1, -1);
-      }
-      
+      parsedFilterData.data = input.slice(beginIndex + 1, -1);
```

Comment:
Question here: the previous code was checking for escaped forward slashes, what about this PR makes that unnecessary now?